.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rosserial_arduino/arduino-cmake"]
+	path = rosserial_arduino/arduino-cmake
+	url = git@github.com:mcgill-robotics/arduino-cmake.git
```

Comment:
Historically, we couldn't do this because bloom didn't support upstreams with submodules. The ticket tracking that appears to be a bit inconclusive as far as the current state of affairs: https://github.com/ros-infrastructure/bloom/issues/217

Could you investigate please, and determine whether this is going to work?

You can try running bloom locally with the following script: https://gist.github.com/mikepurvis/7036293